doc/storagedriver/azure.md

```diff
@@ -10,6 +10,7 @@ The following parameters must be used to authenticate and configure the storage
 * `accountname`: Name of the Azure Storage Account.
 * `accountkey`: Primary or Secondary Key for the Storage Account.
 * `container`: Name of the root storage container in which all registry data will be stored. Must comply the storage container name [requirements][create-container-api].
+* `domain`: (optional) Domain name for the Storage Service. Defaults to `core.windows.net`. For example, `core.chinacloudapi.cn` would be the correct value for Azure in China and `core.usgovcloudapi.net` would be the correct value for Azure Government.
```

Comment:
azureRealm ? It's very close to the notion of a "realm" in Kerberos, which really winds up being the suffix for a variety of names (usually hostnames) that get built.